addons/account/views/account_move_views.xml

```diff
@@ -808,11 +808,11 @@
                                             'readonly': [('state', '!=', 'draft')],
                                        }"/>
                                 <field name="payment_reference"
-                                       attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))], 'readonly': [('state', '!=', 'draft')]}"/>
+                                       attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
```

Comment:
So you give the opportunity to change the reference after having given it to you client, then not being able to match the invoice with the payment? :thinking: 
I don't get the added value; it is computed most of the time anyways.